pronto_mia/lib/core/services/graphql_service.dart

```diff
@@ -83,4 +98,19 @@ class GraphQLService {
       return '';
     }
   }
+
+  bool _isNetworkAvailable(OperationException exception) {
+    if (exception.linkException is NetworkException) {
+      return false;
+    }
+
+    if (exception.linkException is ServerException) {

```

Comment:
Why is this necessary? Is the exception thrown not always a Network exception?

Reply:
No, when I go offline on my end I get a ServerException with an empty parsedResponse property.